src/core/weightedGraph.js

```diff
@@ -48,7 +48,8 @@ export function fromJSON(j: WeightedGraphJSON): WeightedGraph {
  * This delegates to the semantics of Graph.merge and Weights.merge.
  */
 export function merge(ws: $ReadOnlyArray<WeightedGraph>): WeightedGraph {
-  const graph = Graph.merge(ws.map((w) => w.graph));
+  const graphs = ws.map((w) => w.graph);
+  const graph = Graph.merge(graphs);
```

Comment:
Nit: looks like a functionally equivalent, unrelated change

Reply:
Good catch -- added it while debugging something. Will revert.